message/messaging_service.cc

```diff
@@ -1276,4 +1277,17 @@ future<> messaging_service::send_paxos_learn(msg_addr id, clock_type::time_point
         std::move(reply_to), std::move(shard), std::move(response_id), std::move(trace_info));
 }
 
+void messaging_service::register_hint_mutation(std::function<future<rpc::no_wait_type> (const rpc::client_info&, rpc::opt_time_point, frozen_mutation fm, std::vector<inet_address> forward,
+        inet_address reply_to, unsigned shard, response_id_type response_id, rpc::optional<std::optional<tracing::trace_info>> trace_info)>&& func) {
+    register_handler(this, netw::messaging_verb::HINT_MUTATION, std::move(func));
+}
+void messaging_service::unregister_hint_mutation() {
+    _rpc->unregister_handler(netw::messaging_verb::HINT_MUTATION);
+}
+future<> messaging_service::send_hint_mutation(msg_addr id, clock_type::time_point timeout, const frozen_mutation& fm, std::vector<inet_address> forward,
+        inet_address reply_to, unsigned shard, response_id_type response_id, std::optional<tracing::trace_info> trace_info) {
+    return send_message_oneway_timeout(this, timeout, messaging_verb::MUTATION, std::move(id), fm, std::move(forward),
```

Comment:
Should not the verb be `messaging_verb::HINT_MUTATION`?

Reply:
Ha. That's a good catch!

Reply:
Oops... Fixed!

Reply:
Hmmm... I still see `MUTATION` verb. Are you sure you pushed your changes?

Reply:
Well, I did, but I applied the fix to a wrong commit. It should be all in order now.